notification-center/client/react/api-reference.mdx

```diff
@@ -173,7 +173,39 @@ type ObjectWithRoot<T = {}> = T & {
   root: CSSFunctionOrObject;
 };
 ```
-
+<Accordion title="How to use the styles interface?">
+  Using the Styles interface is quite straightforward. All you need to do is identify the property you need to modify to achieve your desired look and pass it as an object in the `styles` prop. 
```

Comment:
I think we should stop using the sentence `quite straightforward` because we are shooting ourselves in the foot. Users still come and tell us they didn't understand our docs.

So you can remove that all sentence and just start with what the reader needs to do @sumitsaurabh927

Reply:
okay